cert-manager/cluster-issuer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+  namespace: kube-system
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: cloud-deployment@forgerock.com
+    # Name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    # Enable the dns01 challenge provider
+    dns01:
+      providers:
+      - name: prod-dns
+        clouddns:
+          project: engineering-devops
+          serviceAccountSecretRef:
+            name: clouddns
```

Comment:
Lets add some README notes or inline comments that briefly explain what is going on here. 
i.e. go to google dns, and do X 
How to create the service account 
what roles the service account needs
etc